pkg/review/commit_review_status.go

```diff
@@ -485,49 +520,50 @@ func formatGoogleSQL(qualifiedTableName *bigqueryio.QualifiedTableName) string {
 // the given GitHub organization, repository, and commit sha. If the commit
 // has no such associated pull requests then an empty slice is returned.
 func GetPullRequestsTargetingDefaultBranch(ctx context.Context, client *githubv4.Client, githubOrg, repository, commitSha string) ([]*PullRequest, error) {
-	var query struct {
-		Repository struct {
-			DefaultBranchRef struct {
-				Name githubv4.String
-			}
-			Object struct {
-				Commit struct {
-					AssociatedPullRequest struct {
-						Nodes    []*PullRequest
-						PageInfo struct {
-							EndCursor       githubv4.String
-							HasNextPage     githubv4.Boolean
-							HasPreviousPage githubv4.Boolean
-							StartCursor     githubv4.String
-						}
-						TotalCount githubv4.Int
-					} `graphql:"associatedPullRequests(first: 100, after: $pageCursor)"`
-				} `graphql:"... on Commit"`
-			} `graphql:"object(oid: $commitSha)"`
-		} `graphql:"repository(owner: $githubOrg, name: $repository)"`
-	}
-
+	query := CommitGraphQlQuery{}
 	pullRequests := make([]*PullRequest, 0, query.Repository.Object.Commit.AssociatedPullRequest.TotalCount)
-	cursor := githubv4.String("")
+	pullRequestCursor := githubv4.String("")
 	for {
-		if err := client.Query(ctx, &query, map[string]any{
-			"githubOrg":  githubv4.String(githubOrg),
-			"repository": githubv4.String(repository),
-			"commitSha":  githubv4.GitObjectID(commitSha),
-			"pageCursor": cursor,
+		if err := client.Query(ctx, &query, map[string]interface{}{
+			"githubOrg":         githubv4.String(githubOrg),
+			"repository":        githubv4.String(repository),
+			"commitSha":         githubv4.GitObjectID(commitSha),
+			"pullRequestCursor": pullRequestCursor,
+			// The initial reviewCursor must be nil and not the empty string "",
+			// unlike the pullRequestCursor.
+			"reviewCursor": (*githubv4.String)(nil),
```

Comment:
Can you just omit this field entirely? That should be the same as setting it to a typed nil.

Reply:
We cannot due to having to annotate the struct with graphql call:
https://github.com/abcxyz/github-metrics-aggregator/blob/7cfa0926f3362c29837a4f0ea7762cba00f5e3e7/pkg/review/commit_review_status.go#L184

If we remove this we get the following error:
```
Variable $reviewCursor is used by anonymous query but not declared
```